arch/arm/soc/nxp_imx/mcimx6x_m4/dts.fixup

```diff
@@ -0,0 +1,98 @@
+#define CONFIG_GPIO_IMX_PORT_1_IRQ_1_PRI	NXP_IMX_GPIO_4209C000_IRQ_1_PRIORITY
+
+#define CONFIG_GPIO_IMX_PORT_2_BASE_ADDRESS	NXP_IMX_GPIO_420A0000_BASE_ADDRESS_0
+#define CONFIG_GPIO_IMX_PORT_2_NAME		NXP_IMX_GPIO_420A0000_LABEL
+#define CONFIG_GPIO_IMX_PORT_2_IRQ_0		NXP_IMX_GPIO_420A0000_IRQ_0
+#define CONFIG_GPIO_IMX_PORT_2_IRQ_0_PRI	NXP_IMX_GPIO_420A0000_IRQ_0_PRIORITY
+#define CONFIG_GPIO_IMX_PORT_2_IRQ_1		NXP_IMX_GPIO_420A0000_IRQ_1
+#define CONFIG_GPIO_IMX_PORT_2_IRQ_1_PRI	NXP_IMX_GPIO_420A0000_IRQ_1_PRIORITY
+
+#define CONFIG_GPIO_IMX_PORT_3_BASE_ADDRESS	NXP_IMX_GPIO_420A4000_BASE_ADDRESS_0
+#define CONFIG_GPIO_IMX_PORT_3_NAME		NXP_IMX_GPIO_420A4000_LABEL
+#define CONFIG_GPIO_IMX_PORT_3_IRQ_0		NXP_IMX_GPIO_420A4000_IRQ_0
+#define CONFIG_GPIO_IMX_PORT_3_IRQ_0_PRI	NXP_IMX_GPIO_420A4000_IRQ_0_PRIORITY
+#define CONFIG_GPIO_IMX_PORT_3_IRQ_1		NXP_IMX_GPIO_420A4000_IRQ_1
+#define CONFIG_GPIO_IMX_PORT_3_IRQ_1_PRI	NXP_IMX_GPIO_420A4000_IRQ_1_PRIORITY
+
+#define CONFIG_GPIO_IMX_PORT_4_BASE_ADDRESS	NXP_IMX_GPIO_420A8000_BASE_ADDRESS_0
+#define CONFIG_GPIO_IMX_PORT_4_NAME		NXP_IMX_GPIO_420A8000_LABEL
+#define CONFIG_GPIO_IMX_PORT_4_IRQ_0		NXP_IMX_GPIO_420A8000_IRQ_0
+#define CONFIG_GPIO_IMX_PORT_4_IRQ_0_PRI	NXP_IMX_GPIO_420A8000_IRQ_0_PRIORITY
+#define CONFIG_GPIO_IMX_PORT_4_IRQ_1		NXP_IMX_GPIO_420A8000_IRQ_1
+#define CONFIG_GPIO_IMX_PORT_4_IRQ_1_PRI	NXP_IMX_GPIO_420A8000_IRQ_1_PRIORITY
+
+#define CONFIG_GPIO_IMX_PORT_5_BASE_ADDRESS	NXP_IMX_GPIO_420AC000_BASE_ADDRESS_0
+#define CONFIG_GPIO_IMX_PORT_5_NAME		NXP_IMX_GPIO_420AC000_LABEL
+#define CONFIG_GPIO_IMX_PORT_5_IRQ_0		NXP_IMX_GPIO_420AC000_IRQ_0
+#define CONFIG_GPIO_IMX_PORT_5_IRQ_0_PRI	NXP_IMX_GPIO_420AC000_IRQ_0_PRIORITY
+#define CONFIG_GPIO_IMX_PORT_5_IRQ_1		NXP_IMX_GPIO_420AC000_IRQ_1
+#define CONFIG_GPIO_IMX_PORT_5_IRQ_1_PRI	NXP_IMX_GPIO_420AC000_IRQ_1_PRIORITY
+
+#define CONFIG_GPIO_IMX_PORT_6_BASE_ADDRESS	NXP_IMX_GPIO_420B0000_BASE_ADDRESS_0
+#define CONFIG_GPIO_IMX_PORT_6_NAME		NXP_IMX_GPIO_420B0000_LABEL
+#define CONFIG_GPIO_IMX_PORT_6_IRQ_0		NXP_IMX_GPIO_420B0000_IRQ_0
+#define CONFIG_GPIO_IMX_PORT_6_IRQ_0_PRI	NXP_IMX_GPIO_420B0000_IRQ_0_PRIORITY
+#define CONFIG_GPIO_IMX_PORT_6_IRQ_1		NXP_IMX_GPIO_420B0000_IRQ_1
+#define CONFIG_GPIO_IMX_PORT_6_IRQ_1_PRI	NXP_IMX_GPIO_420B0000_IRQ_1_PRIORITY
+
+#define CONFIG_GPIO_IMX_PORT_7_BASE_ADDRESS	NXP_IMX_GPIO_420B4000_BASE_ADDRESS_0
+#define CONFIG_GPIO_IMX_PORT_7_NAME		NXP_IMX_GPIO_420B4000_LABEL
+#define CONFIG_GPIO_IMX_PORT_7_IRQ_0		NXP_IMX_GPIO_420B4000_IRQ_0
+#define CONFIG_GPIO_IMX_PORT_7_IRQ_0_PRI	NXP_IMX_GPIO_420B4000_IRQ_0_PRIORITY
+#define CONFIG_GPIO_IMX_PORT_7_IRQ_1		NXP_IMX_GPIO_420B4000_IRQ_1
+#define CONFIG_GPIO_IMX_PORT_7_IRQ_1_PRI	NXP_IMX_GPIO_420B4000_IRQ_1_PRIORITY
+
+#define CONFIG_UART_IMX_UART_1_BASE_ADDRESS	NXP_IMX_UART_42020000_BASE_ADDRESS_0
+#define CONFIG_UART_IMX_UART_1_NAME		NXP_IMX_UART_42020000_LABEL
+#define CONFIG_UART_IMX_UART_1_IRQ		NXP_IMX_UART_42020000_IRQ_0
```

Comment:
Use `CONFIG_UART_IMX_UART_[X]_IRQ_NUM` instead of `CONFIG_UART_IMX_UART_[X]_IRQ`

Reply:
Changed.